installer/pkg/components/server/types.go

```diff
@@ -130,14 +130,17 @@ type WorkspaceGarbageCollection struct {
 }
 
 type GitHubApp struct {
-	Enabled         bool   `json:"enabled"`
-	AppId           int32  `json:"appId"`
-	BaseUrl         string `json:"baseUrl"`
-	WebhookSecret   string `json:"webhookSecret"`
-	AuthProviderId  string `json:"authProviderId"`
-	CertPath        string `json:"certPath"`
-	MarketplaceName string `json:"marketplaceName"`
-	LogLevel        string `json:"logLevel"`
+	Enabled             bool   `json:"enabled"`
+	AppId               int32  `json:"appId"`
+	AppIdPath           string `json:"appIdPath"`
+	BaseUrl             string `json:"baseUrl"`
+	WebhookSecret       string `json:"webhookSecret"`
+	WebhookSecretPath   string `json:"webhookSecretPath"`
+	AuthProviderId      string `json:"authProviderId"`
+	CertPath            string `json:"certPath"`
+	MarketplaceName     string `json:"marketplaceName"`
+	MarketplaceNamePath string `json:"marketplaceNamePath"`
```

Comment:
No self-hosted user will integrate with the GitHub market place. For SaaS we are considering to remove this integration as well. Adding this now does not seem sensical.

Reply:
Ok

Reply:
Yeah, to clarify, the GitHub app is required for managing webhooks for prebuilds.
Installing/authorizing the app in GitHub accounts is now integrated with Gitpod projects.

The GitHub marketplace config should be optional (or managed separately).